### DISCUSSION TITLE
Xbox controller carbon - downgrade not possible

### DISCUSSION THREAD

**Discussion by tzakrzewski-dev** [ndeadly/MissionControl#754] [General]
hello
i try to downgrade my controller xbox one with BLE and it's not possible with computer. 
it will be possible with mission control to update it in order to be compatible with BLE or not

**Comment by ndeadly** [comment]
It's not possible to downgrade the firmware on the Carbon Black controller. The earliest firmware for these ones was already using Bluetooth LE. I'm working on adding support for LE controllers. There are experimental test builds available on my discord server that are capable of connecting them.

**Comment by tzakrzewski-dev** [comment]
Give me your discord link

Teddy zakrzewski
________________________________

**Reply by ndeadly** [reply]
https://discord.gg/gegfNZ5Ucz
